tests/stress/Makefile

```diff
@@ -1,7 +1,10 @@
-test: 
-	go test -v -parallel 10 -timeout 1h
+init:
+	pushd module ; \
+	ln -s -f $$(which terraform-provider-layer0) . ; \
+	terraform init ; \
+	popd ; \
 
-benchmark:
-	go test -v -run NOTHING -bench . -timeout 2h
+test: init
+	go test -v -run NOTHING -bench . -timeout 2h 
```

Comment:
[nit/suggestion] There may be some small amount of worth in adding the `-debug` flag here? When you ran these tests you had expressed some concern about how long they took, and `-debug` should spit out the names of each test case as they're run.

Reply:
ah good call